### ISSUE TITLE
Git Push

### ISSUE THREAD

**Issue by choudhurym** [cs531-f19/discussions#9]
Hello Everyone,

Did anyone got into trouble while doing "git push"? I was able to commit, however, when I tried doing "git push", I am getting the following error. I will really appreciate your help. Thank you.

Muntabir@Tasin MINGW64 ~/discussions (master)
$ git push
remote: Permission to cs531-f19/discussions.git denied to choudhurym.
fatal: unable to access 'https://github.com/cs531-f19/discussions.git/': The requested URL returned error: 403

**Comment by ibnesayeed** [comment]
I think you have cloned the upstream repo (i.e., this repo) where you do not have write access. You are supposed to fork this repo into your account, clone your fork locally, make changes in your local clone, push it back to your fork, and create pull request to this repo. This is a common workflow in contributing to open source projects. The process is illustrated step by step in the Git/GitHub slides I presented last Thursday (link on the course site). Update your progress here or report any issues you have.

**Comment by choudhurym** [comment]
Yes, later I noticed that I copied the URL from this repository and did not take the URL which I forked into my account. Thanks so much, Sawood. It worked now.
